### ISSUE TITLE
[MediaWiki] Migration to VirtualBox

### ISSUE THREAD

**Issue by emenninga** [bitnami/vms#549]
### Describe your issue as much as you can

I've been using a Bitnami locally hosted mediawiki on a Mac since 2014 to store notes, etc. for my writing projects. I have upgraded a few times over the years and it was working fine until yesterday when I (stupidly) upgraded my OS to Ventura (OSX 13.0.1).  After that, my MediaWiki app crashed on launch.  I found reports of the same issue here (with redmine and dolibarr) and was hoping that I could address my issue, too.
However, I couldn't find the same application that I was using before (or were provided as tests by [jotamartos](https://github.com/jotamartos))  Bitnami's site looks like the whole virtual machine implementation has changed and this morning I installed VirtualBox and the mediawiki VM and I have a mediawiki running again. However, I'm not sure how to transfer the contents from the old system to the new one. And all I have is the stackman/machines/mediawiki/ files from my previous installation.
Long story short:  how can I get a working mediawiki application for Mac Ventura so that I can rescue all of my content?
Thanks for any suggestions.

**Comment by emenninga** [comment]
Or, how about providing better migration support from the old bitnami local virtual machine mechanism to the new one?

**Comment by jotamartos** [comment]
Hi,

First of all, let's try to solve the issue with OS X Ventura. 

- Rename the hyperkit binary

```
sudo mv ~/.bitnami/stackman/helpers/hyperkit ~/.bitnami/stackman/helpers/hyperkit.old
```

- Download the latest version of the OS X VM

https://downloads.bitnami.com/files/stacks/mediawiki/1.39.0-2/bitnami-mediawiki-1.39.0-2-r02-osx-x86_64-vm.dmg 

- Launch the OS X VM. You should be able to start the VM and access your data. 

(optional) Once you confirm everything works properly, you can get a DB backup and a copy of your files and move them to a new VM you launch using VirtualBox

https://docs.bitnami.com/general/apps/mediawiki/administration/upgrade/#migrate-the-database-from-an-old-version-to-a-new-one

**Comment by emenninga** [comment]
Hi [jotamartos](https://github.com/jotamartos),  Thank you for your time & response.  I was able to download that version and transition my data to it.  Everything appears to be back.  One question -- I was looking for this app on the website but couldn't find it.  Is it there? I looked for a long time and would have preferred to find it on my own. 
BTW, I am re-evaluating switching to VirtualBox or just finding a less opaque way to store my data. 
Again -- thank you!

**Comment by jotamartos** [comment]
I'm glad to hear that! 

Regarding your question, we dropped support for the native installers on Nov 30th. You can learn more about this in our blog 

https://blog.bitnami.com/2022/11/focusing-on-cloud-native-solutions.html
